lib/actions/ui.js

```diff
@@ -166,3 +171,32 @@ export function windowMove () {
     });
   };
 }
+
+export function openFile (path) {
+  return (dispatch, getState) => {
+    dispatch({
+      type: UI_OPEN_FILE,
+      effect () {
+        stat(path, (err, stats) => {
+          if (!err) {
```

Comment:
For plugin errors and similar both `console.error` and `notify` are used, like here: https://github.com/zeit/hyperterm/blob/master/lib/utils/plugins.js#L75-L76